assets/templates/main.tmpl

```diff
@@ -39,6 +39,10 @@
   <noscript><iframe src="https://www.googletagmanager.com/ns.html?id=GTM-MBCBVQS"
                     height="0" width="0" style="display:none;visibility:hidden"></iframe></noscript>
   <!-- End Google Tag Manager (noscript) -->
+    {{ if (ne .FeatureFlags.HideCookieBanner true)}}
+      {{ template "partials/banners/cookies" . }}
```

Comment:
Believe this should be {{ if not .FeatureFlags.... }}

Reply:
I'm still getting familiar with Go, should the syntax be:
`{{ if not .FeatureFlags.HideCookieBanner true}}`

Reply:
I'm still getting familiar with Go, should the syntax be:
`{{ if not .FeatureFlags.HideCookieBanner true}}`

Reply:
or this:
`{{ if ne FeatureFlags.HideCookieBanner true }}`